app/models/cmr.rb

```diff
@@ -506,12 +506,9 @@ def self.collection_search(free_text, provider, provider_list, curr_page="1", pa
       #setting the provider params
       if provider == ANY_DAAC_KEYWORD
         base_options['provider'] = provider_list + ['LARC_ASDC', 'LARC'] +
-          ['NSIDCV0', 'NSIDC_ECS'] + ['GHRC', 'GHRC_CLOUD', 'LANCEAMSR2']
+          ['NSIDCV0', 'NSIDC_ECS'] + ['GHRC', 'GHRC_CLOUD', 'GHRC_DAAC', 'LANCEAMSR2']
```

Comment:
Shouldn't this include ASDC NSIDC and GHRC as well?

Reply:
Now included